### PR TITLE
Add 101-analysis-services-create

### PR DESCRIPTION
**Pull request by minghan**
Add quickstart template for creating an Azure Analysis Services resource.

### CODE REVIEW

File: 101-analysis-services-create/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      "defaultValue": {},
+      "metadata": {
+        "description": "The inbound firewall rules to define on the server. If not specified, firewall is disabled."
+      }
+    },
+    "backupBlobContainerUri": {
+      "type": "string",
+      "defaultValue": "",
+      "metadata": {
+        "description": "The URI to an Azure Blob Storage container. Required only if you intend to use the backup/restore functionality. See https://docs.microsoft.com/en-us/azure/analysis-services/analysis-services-backup"
```
Comment (ziyuanms) [inline 101-analysis-services-create/azuredeploy.json]: SAS URI

File: 101-analysis-services-create/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "serverName": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of the Azure Analysis Services server to create. Server name must begin with a letter, be lowercase alphanumeric, and between 3 and 63 characters in length."
```
Comment (bmoore-msft) [inline 101-analysis-services-create/azuredeploy.json]: does this need to be globally unique?

File: 101-analysis-services-create/azuredeploy.json (hunk 1 above)
Comment (bmoore-msft) [reply 101-analysis-services-create/azuredeploy.json]: yes, need more info here?  Does it need to be public? can it be sas? and if so what are the perms?

File: 101-analysis-services-create/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        "description": "The URI to an Azure Blob Storage container. Required only if you intend to use the backup/restore functionality. See https://docs.microsoft.com/en-us/azure/analysis-services/analysis-services-backup"
+      }
+    }
+  },
+  "resources": [
+    {
+      "apiVersion": "2017-08-01",
+      "name": "[parameters('serverName')]",
+      "type": "Microsoft.AnalysisServices/servers",
+      "location": "[resourceGroup().location]",
```
Comment (bmoore-msft) [inline 101-analysis-services-create/azuredeploy.json]: Add location parameter - see: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#parameters

File: 101-analysis-services-create/azuredeploy.parameters.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "serverName": {
+      "value": "GEN-UNIQUE"
+    },
+    "skuName": {
```
Comment (bmoore-msft) [inline 101-analysis-services-create/azuredeploy.parameters.json]: move any params that are not GEN values to defaultValues in the template

File: 101-analysis-services-create/azuredeploy.json (hunk 2 above)
Comment (minghan) [reply 101-analysis-services-create/azuredeploy.json]: No, unique per region. Will update.

**Pull request merged**